### ISSUE TITLE
Reverse horizontal scrolling direction [enhancement]

### ISSUE THREAD

**Issue by danielmanciu** [noah-nuebling/Mac-Mouse-Fix#989]
## Description
Add a new option "Reverse Horizontal Direction", similar to "Reverse Direction" inside the "Scrolling" menu, but only operating on the horizontal scroll axis.

## Rationale
I recently purchased a Logitech MX Master 3S mouse and it has a horizontal scrolling wheel. Contrary to vertical scrolling, Apple implemented horizontal scrolling to be as expected. This means that if I enable Mac Mouse Fix's "Reverse Direction", the vertical scrolling will be as I need it, but horizontal will be reversed.

## Additional info
–  Implementing this change would involve changing the current "Reverse Direction" to be "Reverse Vertical Direction" and only affect the vertical scroll axis.

**Comment by hollie** [comment]
+1 for this enhancement. I just unboxed my Logitech MX Master, am under impressed by the Logitech bundled software and switched to mac-mouse-fix. The horizontal scroll wheel direction reverse is something I would use too!

Best regards,
 Hollie

**Comment by chuqbach** [comment]
Same with me, older version of Mac Mouse Fix doesn't reverse the horizontal scrolling, but the current version reverse both vertical and horizontal, which is really annoying.

**Comment by matteoveglia** [comment]
Yeah +1 here too, I just ditched my MX Master 3S (yet another left click button failure) for a Rapoo MT760 which also has a horizontal scroll wheel

**Comment by ernest-heh** [comment]
+1 as well

**Comment by remondulos** [comment]
+1 for me as well

**Comment by Toastmeister69** [comment]
im late to this, but i use this one https://pilotmoon.com/scrollreverser/
